hugo/content/news/2014-03-12-mumble-for-ios-1-3-0.md

```diff
@@ -17,4 +17,4 @@ As always, it’s available on the [App Store][1]. And don’t forget — if you
 The Mumble team
 
  [1]: https://itunes.apple.com/us/app/mumble/id443472808?mt=8
- [2]: http://blog.mumble.info/mumble-for-ios-1-2-3-is-buggy-when-running-on-ios-7-1/
+ [2]: https://www.mumble.info/news/mumble-for-ios-1.2.3-is-buggy-when-running-on-ios-7.1/
```

Comment:
Are you sure that the link has to be adjusted to use dots instead of dashes? :thinking:

Reply:
I have simply copied the link directly from the browser and changed `blog` to `news`, so I assume: yes that is correct and necessary.

Reply:
Just checked: So the old link is also working but it seems to be a redirect to the one using the actual dots :point_up: